src/cdk/private/testing/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tools:defaults.bzl", "ng_module")
+
+ng_module(
```

Comment:
I see that there is an `@Injectable` in the package here, but should we also try using `ts_library`? It feels a bit confusing as there is no NgModule in the package right?